<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FavoriteWidget</class>
 <widget class="QWidget" name="FavoriteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="favwidget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>981</width>
         <height>51</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>981</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="musicpushButton">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>歌曲</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="albumpushButton">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>专辑</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="listpushButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>歌单</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mvpushButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>MV</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="favwidget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>51</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>我喜欢</string>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>981</width>
         <height>501</height>
        </rect>
       </property>
       <widget class="QWidget" name="musicpage">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>981</width>
           <height>51</height>
          </rect>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>551</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="PlaypushButton">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>播放全部</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DownloadpushButton">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>下载</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MultipushButton">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>批量操作</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>770</x>
            <y>10</y>
            <width>175</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QRowTableView" name="musictableWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>961</width>
           <height>441</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="albumpage">
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>130</y>
           <width>26</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="listpage">
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>170</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="mvpage">
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>180</y>
           <width>86</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRowTableView</class>
   <extends>QTableView</extends>
   <header>qrowtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
